server/src/services/EmailService.js

```diff
@@ -0,0 +1,75 @@
+const Schmervice = require('@hapipal/schmervice')
+const Hapi = require('@hapi/hapi')
+const Nodemailer = require('nodemailer')
+const JWT = require('jsonwebtoken')
+const AWS = require('aws-sdk')
+const debug = require('debug')('apphub:server:service:EmailService')
+
+const FROM_INFO = {
+    name: 'DHIS2 App Hub',
+    address: 'apphub@dhis2.org',
+}
+class EmailService extends Schmervice.Service {
+    constructor(server, schmerviceOptions, serviceOptions) {
+        super(server, schmerviceOptions)
+
+        if (!serviceOptions.transport) {
+            server.logger.warn(
```

Comment:
I don't think `sendmail` as a fallback transport is what we want - it might be better to use a `console.log`/`debug` only transport for development that just prints out the email contents while for production we should enforce `serviceOptions.transport`.

Reply:
Here's a minimal development transport:

```
let transport = {
    name: 'developmentTransport',
    send: (mail, callback) => {
        const input = mail.message.createReadStream()
        console.log('Sent email:')
        input.pipe(process.stdout)
        input.on('end', function () {
            callback(null, true)
        })
    }
};
```

Reply:
Ive replacted it with a JSONTransport for testing.